docs/GoTopButton.md

```diff
@@ -0,0 +1,54 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Go Top Button
+
+`GoTopButton` is a store block that redirects users to the top of the page when clicked on. This Component can be imported and used by any VTEX app.
+
+![image](https://user-images.githubusercontent.com/28419764/77644893-9238af80-6f40-11ea-8ceb-7355d0c12686.png)
+
+## Configuration
+
+1. Import the `vtex.store-component` app to your theme's dependencies in the manifest.json;
+
+```json
+ "dependencies: {
+    "vtex.store-components": "3.x"
+  }
+```
+
+2. Add the `go-top-button` block into a store template of your choosing. In the example, it will be added to the home page:
+
+```diff
+  "store.home": {
+    "blocks": [
++     "go-top-button",
+    ]
+  },
+```
+
+3. Then, declare the go-top-button block using its props stated in the table below:
+
+```diff
+  "store.home": {
+    "blocks": [
+     "go-top-button",
+    ]
+  },
++  "go-top-button":{
```

Comment:
I think `back-to-top` is a more widespread name for this kind of component
(look at the search term and the search results on the screenshot below)

![Screen Shot 2020-03-31 at 14 06 47](https://user-images.githubusercontent.com/5691711/78054794-f9979a80-7358-11ea-8967-e1be8a89b911.png)